### PR TITLE
(FACT-2722) Add disks fact for Solaris

### PR DESCRIPTION
**Pull request by Filipovici-Andrei**

### CODE REVIEW

**Comment by Filipovici-Andrei** [comment]
jenkins please test this on all

**Comment by Filipovici-Andrei** [comment]
Sure @igalic. That's a reasonable request.

File: lib/facter/resolvers/load_averages.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 module Facter
   module Resolvers
     class LoadAverages < BaseResolver
+      @log = Facter::Log.new(self)
```
Comment (IrimieBogdan) [inline lib/facter/resolvers/load_averages.rb:6]: You don't need to define a logger, the base resolver will automatically create one for you. You can use it directly via 
`log.debug(...)`

File: lib/facter/resolvers/solaris/disks.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+            @log.debug('loading disks info')
+
+            kstat_output = Facter::Core::Execution.execute('/usr/bin/kstat sderr', logger: log)
+            return if kstat_output.empty?
+
+            @fact_list[fact_name] = parse(kstat_output)
+          end
+
+          def parse(input)
```
Comment (IrimieBogdan) [inline lib/facter/resolvers/solaris/disks.rb:29]: `input` is generic for a variable name, maybe `kstat_output `

**Pull request merged**